api/models.py

```diff
@@ -12,7 +12,7 @@ class UserProfile(models.Model):
 	)
 
 	user = models.OneToOneField(User) 
-	#id = models.AutoField(primary_key=True) 
+	id = models.AutoField(primary_key=True) 
```

Comment:
You don't need this - it's implied by Django - https://docs.djangoproject.com/en/1.7/topics/db/models/#automatic-primary-key-fields